envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -22,6 +22,7 @@ class SnapshotProperties {
     var dynamicListeners = ListenersFactoryProperties()
     var enabledCommunicationModes = EnabledCommunicationModes()
     var shouldSendMissingEndpoints = false
+    var hostHeaderRewriting = HostHeaderRewritingProperties()
```

Comment:
Wouldn't it fit better in EgressProperties? We have there `handleInternalRedirect` which is also a feature set per an outgoing dependency.

Reply:
I had trouble finding right place for it so I put it here, but you are right and it will fit better in EgressProperties I will move it there